src/components/Modal/Modal.module.css

```diff
@@ -96,30 +94,38 @@
     }
 
     .content {
-      width: 100%;
+      min-width: 0;
```

Comment:
Do you think it's important to know why this part was included?
_(avoid content or aside overflowing)_

Reply:
Sure. Let me know if the comment added is ok.

Reply:
reading it made it clear to me without previous knowledge, so I guess it's fine :)